### PR TITLE
[Test] Unlogged customer is unable to buy product without shipping option

### PR DESCRIPTION
**Pull request by yellowee**
I want to merge this change because it's adding a checkout e2e test: Should not be possible to buy product without shipping option CORE_0106

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are either absent or guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migrations are either absent or optimized for zero downtime
* [ ] The changes are covered by test cases

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -11,5 +11,8 @@
         "preorders",
         "undiscounted"
     ],
-    "python.linting.enabled": true
+    "python.linting.enabled": true,
+    "[python]": {
```
Comment (korycins) [inline .vscode/settings.json:15]: I am not convinced that we should modify the default vscode settings. And if we really want to do that, we should do that in separate PR

File: .vscode/settings.json [hunk 2]
```diff
@@ -11,5 +11,8 @@
...
+        "editor.defaultFormatter": "ms-python.black-formatter"
+    }
```
Comment (IKarbowiak) [inline .vscode/settings.json:17]: These changes are on purpose?

File: .vscode/settings.json (hunk 1 above)
Comment (yellowee) [reply .vscode/settings.json]: Changes reverted by amending [current commit](https://github.com/saleor/saleor/pull/13325/commits/3bc631a7eb5669670648eba18e187c82c03d81b6)

File: .vscode/settings.json (hunk 2 above)
Comment (yellowee) [reply .vscode/settings.json]: Changes reverted by amending [current commit](https://github.com/saleor/saleor/pull/13325/commits/3bc631a7eb5669670648eba18e187c82c03d81b6)

**Comment by yellowee** [comment]
Closing in favour of https://github.com/saleor/saleor/pull/13328
